econml/inference.py

```diff
@@ -949,7 +893,8 @@ def pvalue(self, value=0):
             the corresponding singleton dimensions in the output will be collapsed
             (e.g. if both are vectors, then the output of this method will also be a vector)
         """
-        return min((self.pred_dist < value).sum(), (self.pred_dist > value).sum()) / self.pred_dist.shape[0]
+        return np.minimum((self.pred_dist <= value).sum(axis=0),
```

Comment:
The pvalue calculation has been fixed for empirical bootstrap when point=0 and std=0, but for NormalInference, if point=0 and std=0, we still return a NaN instead of 1 for p-value.

Reply:
Fixed edge cases and added a test.